server/server.py

```diff
@@ -78,6 +49,42 @@ def hello():
     return jsonify(top.current_user)
 
 
+@app.route("/record/<record_id>", methods=('GET', 'PUT'))
+@cross_origin(headers=["Content-Type", "Authorization"])
+@cross_origin(headers=["Access-Control-Allow-Origin", "*"])
+def record(record_id):
+    jwt = request.headers.get("Authorization", b'')
+    filename = os.path.join(HERE, "records", "{record_id}.json".format(
+        record_id=os.path.basename(record_id)))
+    new = True
+    if os.path.exists(filename):
+        new = False
+        with open(filename, 'r') as f:
+            record = json.load(f)
+            author = record["author"]
```

Comment:
we could get rid of `new` and just have `record = None`. And use `record['author']` instead of `author`

(just in the seek of brevity)